### PR TITLE
Context menu (iOS 13)

### PR DESCRIPTION
**Pull request by buh**
Show message actions with the context menu from iOS 13 instead of ActionSheet.

### CODE REVIEW

File: Sources/UI/Chat/Chat View Controller/ChatViewController.swift [hunk 1]
```diff
@@ -31,6 +31,16 @@ open class ChatViewController: ViewController, UITableViewDataSource, UITableVie
         return .all
     }
     
+    /// Message actions (see `MessageAction`).
+    @available(iOS 13, *)
+    public lazy var useContextMenuForActions = defaultUseContextMenuForActions
```
Comment (b-onc) [inline Sources/UI/Chat/Chat View Controller/ChatViewController.swift]: Why do we need 2 variables? Why can't we just use the computed one?
```swift
open var useContextMenuForActions: Bool { true }
```

File: Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift [hunk 2]
```diff
@@ -297,3 +298,136 @@ extension ChatViewController {
...
+                let message = self.channelPresenter?.items[indexPath.row].message else {
+                return nil
+            }
+            
+            let locationInView = tableView.convert(point, to: cell)
+            return self.createActionsContextMenu(from: cell, for: message, locationInView: locationInView)
+        }
+    }
+    
+    public func defaultActionsContextMenu(from cell: UITableViewCell, for message: Message, locationInView: CGPoint) -> UIMenu? {
```
Comment (b-onc) [inline Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift]: Why do we need this function? Why can't this code be in `createActionsContextMenu`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Custom reactions. [#113](https://github.com//issues/113)
 - A new [cumulative reaction type](https://getstream.io/chat/docs/send_reaction/?language=js#cumulative-clap-reactions) (like claps in Medium). [#113](https://github.com//issues/113)
 - Custom fields for reactions. [#113](https://github.com//issues/113)
+- Message actions with a context menu from iOS 13.
```
Comment (b-onc) [inline CHANGELOG.md]: Can we add 

[#115](https://github.com/GetStream/stream-chat-swift/pull/115/) <- rendered markdown

 to this entry?

File: Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift (hunk 2 above)
Comment (buh) [reply Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift]: It's a lot of code there and it's better to move it in a separate extension.

Comment (b-onc) [reply Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift]: Do we need to have it in `ChatViewController.swift`? Can't we put `createActionsContextMenu` here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Custom reactions. [#113](https://github.com//issues/113)
 - A new [cumulative reaction type](https://getstream.io/chat/docs/send_reaction/?language=js#cumulative-clap-reactions) (like claps in Medium). [#113](https://github.com//issues/113)
 - Custom fields for reactions. [#113](https://github.com//issues/113)
+- Message actions with a context menu from iOS 13. [#115](https://github.com//issues/115)
```
Comment (b-onc) [inline CHANGELOG.md]: Link is broken! :D Missing `GetStream/stream-chat-ios`

Comment (buh) [reply CHANGELOG.md]: oh!

Comment (buh) [reply CHANGELOG.md]: oh! done!

**Pull request merged**